src/Cropper.tsx

```diff
@@ -218,6 +220,11 @@ class Cropper extends React.Component<CropperProps, State> {
     if (prevProps.video !== this.props.video) {
       this.videoRef.current?.load()
     }
+
+    const objectFit = this.getObjectFit();
```

Comment:
We could not catch this through props since it's not the _props.objectFit_ changes

What is being changed is the result of _getObjectFit()_ method that calculates how to properly align the image inside the container (should horizontal-cover or vertical-cover be applied)

- To determine which one should be applied the image should already be presented on the layout to get its sizes (that's why requires re-render)
- Since we already calculated the sizes and props haven't changed on the second render run, the componentDidUpdate() doesn't call computeSIzes() method

Thats why on getObjectFit() result change we need a callback for computeSizes() and re-render (on state change) to apply the changes